lib/tlscontext.c

```diff
@@ -848,6 +854,50 @@ tls_context_setup_sigalgs(TLSContext *self)
   return TRUE;
 }
 
+#if SYSLOG_NG_HAVE_DECL_SSL_CONF_CMD
+static gboolean
+_apply_cmd_context(TLSContext *self)
+{
+  gboolean result = TRUE;
+
+  if (self->conf_cmds_list) {
```

Comment:
do we need this if? I think the for should handle the case where the head of the list is NULL.

Reply:
definitely, it's a leftover again, thanks for pointing that out!